data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/OrderedAsyncExecutor.java

```diff
@@ -83,4 +83,8 @@ public void stop() {
     this.isStopped = true;
     this.queue.clear();
   }
+
+  public Queue<Supplier<Future<?>>> getQueue(){
+    return this.queue;
+  }
```

Comment:
I would expose only the depth/length as a method here.

https://betterprogramming.pub/demeters-law-don-t-talk-to-strangers-87bb4af11694